tests/sql/parsing/queries/case_when_null.sql

```diff
@@ -3,7 +3,7 @@ SELECT CASE
          ELSE store.item_sk
        END              item_sk,
        CASE
-         WHEN web.d_date IS NOT NULL THEN web.d_date
+         WHEN NOT web.d_date IS NULL THEN web.d_date
```

Comment:
`IS NOT NULL` is more SQL-ish

Reply:
I think this was due to a small bug in the build. I believe @CircArgs is planning a fast follow-up to fix this.